src/main/java/mil/dds/anet/database/PositionDao.java

```diff
@@ -577,6 +596,11 @@ public int mergePositions(Position winner, Position loser) {
     updateM2mForMerge("authorizationGroupPositions", "authorizationGroupUuid", "positionUuid",
         winnerUuid, loserUuid);
 
+    // Update organizationResponsiblePositions
+    if (AuthUtils.isAdmin(user)) {
```

Comment:
Is this correct? If a superuser merges two positions, what should happen to the responsibleOrganizations?